approaches/bulk-data-loading/src/GqlMutations.js

```diff
@@ -0,0 +1,85 @@
+export class GqlMutations {
+
+    static createInventoryListQuery() {
```

Comment:
This query should be doable with the built-in client functions
`vendiaClient.entities.inventory.list()`

Reply:
Done